runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -334,12 +339,14 @@ public class CrypticClue extends ClueScroll implements NpcClueScroll, ObjectClue
 	private final String text;
 	private final String npc;
 	private final int objectId;
-	@Nullable
-	private final WorldPoint location;
 	private final String solution;
 	@Nullable
 	private final String questionText;
 
+	@Nullable
```

Comment:
This seems like it should be `Function<ClueScrollPlugin, @Nullable WorldPoint>`, not `@Nullable Function<ClueScrollPlugin, @Nullable WorldPoint>`. As-is, this introduces another field which must be null-checked

Reply:
I think either way is ok. It is +1 extra null check in getLocation() below which seems fine.